hoss/hoss_config.json

```diff
@@ -357,6 +371,19 @@
         }
       ],
       "_Description": "Ensure the latitude and longitude dimension variables know their associated grid_mapping variable."
+    },
+    {
+      "Applicability": {
+        "Mission": "ICESat2",
+        "ShortNamePath": "ATL16",
+        "Variable_Pattern": "/spolar_(asr_obs_grid|lorate_blowing_snow_freq)"
+      },
+      "Attributes": [
+        {
+          "Name": "grid_mapping",
+          "Value": "crs_latlon: spolar_grid_lat crs_latlon: spolar_grid_lon"
+        }
+      ]
```

Comment:
This entry addresses what we think is a mistake in the ATL16 v004 data for the two indicated variables, where the `grid_mapping` attribute references north polar coordinates instead of south polar coordinates. Without this attribute addition all the subsets that include either of these variables would fail due to the existing bug captured in [DAS-1804](https://bugs.earthdata.nasa.gov/browse/DAS-1804)

Reply:
For the record, that grid_mapping value is referencing the crs_latlon grid-mapping variable for both horizontal dimensions (lat, lon), here explicitly identified in the grid_mapping reference value.  A simpler entry would be, I think, to specify "crs_latlon" as a grid_mapping variable that implicitly applies to all dimensions.  This posted override is in keeping with the other ATL16 grid_mapping reference attributes that explicitly call out the dimensional applicability.

Reply:
Totally agree that `crs_latlon` is simpler, but it seems best to match the format of the other entries for `grid_mapping` that NSIDC is using (which is still a valid format per the CF Conventions).

Reply:
I almost wonder if HOSS should be a bit less "diligent" in reading those attributes and instead use other heuristics (likely already in use) for identifying the relevant horizontal dimensions for grid mapping references.  They seem superfluous in this case, and then they got them wrong!

Reply:
Is it HOSS reading those attributes, or varinfo? I saw that varinfo specifically checks the `grid_mapping` attribute if one exists.

Reply:
Good point @lyonthefrog - it is `VarInfoFromDmr` doing the reading. And I think we want to keep `earthdata-varinfo` as aligned to the standard (CF Conventions) as possible. This format of representation is valid per the CF Conventions, so I think we're okay. The only intrinsic error here was that the metadata was incorrect in the file. (Quirky Data!)